ReactViewResources/Loader.ts

```diff
@@ -163,15 +172,15 @@ export function loadPlugins(plugins: string[][], mappings: Dictionary<string>):
 export function loadComponent(
     componentName: string,
     componentSource: string,
+    dependencySources: string[],
+    cssSources: string[],
+    originalSourceFolder: string,
     componentNativeObjectName: string,
-    baseUrl: string,
-    cacheInvalidationSuffix: string,
     maxPreRenderedCacheEntries: number,
     hasStyleSheet: boolean,
     hasPlugins: boolean,
     componentNativeObject: Dictionary<any>,
-    componentHash: string,
-    mappings: Dictionary<string>): void {
+    componentHash: string): void {
 
     function getComponentCacheKey(propertiesHash: string) {
         return componentSource + "|" + propertiesHash;
```

Comment:
Just a heads-up: this componentSource is now an URL: e.g. `webview:\\Example\Generated\Example.view.js`. Previously it was something like `Example\Generated\Example.view`. Do you foresee any problems due to this? It still has two uses below (`localStorage.getItem(componentSource)` and `localStorage.setItem(componentSource)`;

Reply:
If the previous approach is better, I can provide one more parameter

Reply:
thats ok, I don't see any problems since the domain is always the same